src/graphql/generated.ts

```diff
@@ -2376,8 +2475,8 @@ export const GetShoppingCartOfUserDocument = gql`
     }
 `
 export const AddItemToShoppingCartDocument = gql`
-    mutation addItemToShoppingCart($input: AddShoppingCartItemInput!) {
-        addShoppingcartItem(input: $input) {
+    mutation addItemToShoppingCart($input: CreateShoppingCartItemInput!) {
```

Comment:
I would recommend to also rename your function names

Reply:
I thought about that, too. However, I'm a bit hesitant since I named many create operations "addX" and I would have to touch a lot of files. I would like to keep this PR as small as possible because I want to return to the checkout feature implementation as soon as possible. Idk, maybe I should reconsider the renaming in a little while.